packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -958,7 +958,7 @@ class ScrollableBaseNative extends Component {
 		this.scrollStopJob.start();
 	}
 
-	startScrollJob = new Job(this.scrollOnScroll, 16);
+	startScrollJob = new Job(this.scrollOnScroll);
```

Comment:
I think this is a good change but perhaps not for the reason you intended it. When testing a VirtualListNative sample, I noticed that including a timeout along with my proposal to include `stop()` in `startRafAfter` resulted in long delays for rendering items because each scroll event cancelled the previous RAF so items weren't rendered until scrolling stopped.